user_info_helper.go

```diff
@@ -55,3 +55,13 @@ func (x respUserGet) intoUserInfo() UserInfo {
 		QRCodeURL:   x.QRCodeURL,
 	}
 }
+
+func (x respUsersByDept) intoUserInfos() []*UserInfo {
+	var users []*UserInfo
```

Comment:
The outgoing array has same length as incoming (self) array, so you can pre-allocate for clarity (and dubious performance gain).

```suggestion
	users := make([]*UserInfo, len(x.Users))
```